handler.go

```diff
@@ -104,6 +110,13 @@ func (h *langHandler) lint(uri DocumentURI) ([]Diagnostic, error) {
 	return diagnostics, nil
 }
 
+func max(a, b int) int {
```

Comment:
go 1.21 has max/min builtin, sadly golang allows shadowing of builtins.

can we bump golang go 1.21 and use builtin ?